package.json

```diff
@@ -16,7 +16,7 @@
     "hexo-generator-feed": "^1.1.0",
     "hexo-generator-tag": "^0.2.0",
     "hexo-renderer-ejs": "^0.2.0",
-    "hexo-renderer-marked": "^0.2.10",
+    "hexo-renderer-marked": "ngokevin/hexo-renderer-marked#9497bdd",
```

Comment:
did you see [this PR](https://github.com/hexojs/hexo-renderer-marked/pull/24/files)?